### DISCUSSION TITLE
What's required for corporate contributions to ArgoCD?

### DISCUSSION THREAD

**Discussion by andrii-korotkov-verkada** [argoproj/argo-cd#18127] [Q&A] [answered]
Hey. I hope your day is going well. I have a question about requirements to have corporate employees contributing to https://github.com/argoproj/argo-cd. The information is inconsistent between this project and the parent project. For ArgoCD, contributions file directs to https://argo-cd.readthedocs.io/en/latest/developer-guide/code-contributions/, which doesn't mention signing any agreement in a current version. But the parent project has a number of docs, like corporate CLA https://github.com/argoproj/argoproj/blob/main/community/Argo%20Corporate%20CLA.pdf. Hence, I'm curious what's going to apply and what are the specific requirements.

**Comment by crenshaw-dev** [comment]
As far as I know, all we require is a "sign off" on each commit. I believe the linked CLA is no longer used. I'll get confirmation.

**Reply by agilgur5** [reply, marked as answer]
Yes, CLA was used some years ago but then Argo moved to DCO per https://github.com/argoproj/argoproj/pull/18 & https://github.com/argoproj/argoproj/pull/16
